### PR TITLE
Feature/Update-of-HTS-indicators-on-aggregate-tables

### PR DESCRIPTION
**Pull request by nobert-mumo**

### CODE REVIEW

File: Scripts/REPORTING/load_aggregate_hts_pns_knowledge_HIV_status.sql [hunk 1]
```diff
@@ -125,14 +121,23 @@ SELECT
...
-	count(PatientPk)ContactElicited, 
-	sum(ContactPatientPk)ContactTested,
-	sum(FinalResult)Positive, 
-	sum(Linked)Linked,
-	sum(KP)KnownPositive, 
-	sum(NewNegatives)NewNegatives,
-	sum(NewPositives)NewPositives, 
-	sum(UnknownStatus)UnknownStatus
-from combined
-where FinalTestResult='Positive'
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_aggregate_hts_pns_knowledge_HIV_status.sql:137]: @cbrianbet / @meggmumbi just consulting here: what was the reason for filtering for only 'Positive' results here again?

Comment (cbrianbet) [reply Scripts/REPORTING/load_aggregate_hts_pns_knowledge_HIV_status.sql:137]: Hey @nobert-mumo we were borrowing from the old pentaho etl scripts which had this line

Comment (nobert-mumo) [reply Scripts/REPORTING/load_aggregate_hts_pns_knowledge_HIV_status.sql:137]: ah okay, @Marymary-dev, do you know why we used to filter for only positive? would it affect the cascades ?

Comment (Marymary-dev) [reply Scripts/REPORTING/load_aggregate_hts_pns_knowledge_HIV_status.sql:137]: This must have been required for the indicator for HTS POS only

Comment (nobert-mumo) [reply Scripts/REPORTING/load_aggregate_hts_pns_knowledge_HIV_status.sql:137]: sure @Marymary-dev , so we should do away with the filter in these case ?

**Review by meggmumbi — APPROVED** [review]

**Review by cbrianbet — APPROVED** [review]

File: Scripts/REPORTING/load_aggregate_hts_pns_knowledge_HIV_status.sql (hunk 1 above)
Comment (Marymary-dev) [reply Scripts/REPORTING/load_aggregate_hts_pns_knowledge_HIV_status.sql:137]: @cbrianbet  which dashboard was this query fetching for?

**Review by Marymary-dev — APPROVED** [review]

File: Scripts/REPORTING/load_aggregate_hts_pns_knowledge_HIV_status.sql (hunk 1 above)
Comment (cbrianbet) [reply Scripts/REPORTING/load_aggregate_hts_pns_knowledge_HIV_status.sql:137]: @Marymary-dev 
image

**Comment by Marymary-dev** [comment]
I get it so that query was only to fetch for the Positives only, makes sense now , thanks.

**Pull request merged**